test/js/specs/happy.js

```diff
@@ -0,0 +1,19 @@
+const path = require('path')
+import {describe, it} from '../utils/mochaw'
+
+const options = {
+  pageObjects: ['DocumentSelection', 'Welcome', 'DocumentUpload']
+}
+
+describe('Happy Paths',options, ({driver,pageObjects}) => {
+  const {documentSelection, welcome, documentUpload} = pageObjects
+
+  it('should upload a file', async () => {
+    console.log("testing")
+    driver.get('https://localhost:8080/?async=false')
```

Comment:
We could extract this into a function that accepts query strings as an argument so we can make it work with multiple query strings and reuse it in other tests

Reply:
I would leave that complexity for the next person that that adds a second test, but good idea